smartapps/konnected-io/konnected-security-connect.src/konnected-security-connect.groovy

```diff
@@ -269,21 +272,35 @@ def discoverySearchHandler(evt) {
   } else {
     log.debug "Discovered new device $event.mac"
     devices[ssdpUSN] = event
+    discoveryVerify(event)
```

Comment:
Simple optimization to verify the device immediately after it's discovered, instead of waiting for the next refresh interval.